lib/engine/game/g_1822_pnw/entities.rb

```diff
@@ -414,7 +414,7 @@ module Entities
                 when: 'track',
                 count: 1,
                 closed_when_used_up: false,
-                hexes: %w[B15 C14 D15 E16 F17 H17 I18 K16 L15],
+                hexes: %w[A16 B15 C14 D15 E16 F17 H17 I18 K16 L15 M14 N13],
```

Comment:
Also should include G18 and J17 right?

Reply:
thanks !

Can be squashed+merged I think